src/data/index.js

```diff
@@ -1820,4 +1820,52 @@ export default [
       'Bloomberg',
     ],
   },
+  {
+    id: 164,
+    name: 'Longest Word in Dictionary',
+    url: 'https://leetcode.com/problems/longest-word-in-dictionary/',
+    pattern: 'Trie',
+    difficulty: 'Easy',
+    companies: [],
+  },
+  {
+    id: 165,
+    name: 'Maximum XOR of Two Numbers in an Array',
+    url: 'https://leetcode.com/problems/maximum-xor-of-two-numbers-in-an-array',
+    pattern: 'Trie',
+    difficulty: 'Medium',
+    companies: [],
+  },
+  {
+    id: 166,
+    name: 'Concatenated Words',
+    url: 'https://leetcode.com/problems/concatenated-words/',
+    pattern: 'Trie',
+    difficulty: 'Hard',
+    companies: [],
+  },
+  {
+    id: 167,
+    name: 'Prefix and Suffix Search',
+    url: 'https://leetcode.com/problems/prefix-and-suffix-search/',
+    pattern: 'Trie',
+    difficulty: 'Hard',
+    companies: [],
```

Comment:
The companies for `'Prefix and Suffix Search'` are: `'Google', 'Uber', 'Facebook'`

![image](https://user-images.githubusercontent.com/13009507/82843268-a9e1d500-9eaa-11ea-839f-7ff86af07ae7.png)